packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1179,6 +1186,8 @@ export class MergeTree {
 	 */
 	public referencePositionToLocalPosition(
 		refPos: ReferencePosition,
+		// Note: this is not `this.collabWindow.currentSeq` because we want to support resolving local reference positions to positions
```

Comment:
Seems like it might be worth reorganizing the order of eventing and updating sequence numbers, as I could see this being the source of bugs. Doesn't seem like our test coverage is super great in this area either, though the weirdness shouldn't be exposed to our customers, so probably lower on the priority list of merge-tree improvements we have planned.